.sourcery-templates/DerivePartialIsos.stencil

```diff
@@ -0,0 +1,41 @@
+import ApplicativeRouter
+import Either
+import Prelude
+
+{% for enum in types.implementing.DerivePartialIsos|enum %}
+  {% for case in enum.cases %}
+
+    {% if case.hasAssociatedValue %}
+
+      extension PartialIso where A == (
+        {% for value in case.associatedValues %}
+          {% if value.type %}
+            {{ value.type.globalName }}{% if value.typeName.isOptional %}?{% endif %}
```

Comment:
there's a lot of hacky stuff happening in this stencil cause apparently sourcery and/or stencil doesnt have great nested type support. see more here https://github.com/krzysztofzablocki/Sourcery/issues/398